jsonapi/fixtures_test.go

```diff
@@ -301,22 +387,24 @@ func (p AnotherPost) GetReferencedIDs() []ReferenceID {
 
 type ZeroPost struct {
 	ID    string     `json:"-"`
+	LID   string     `json:"-"`
 	Title string     `json:"title"`
 	Value zero.Float `json:"value"`
 }
 
-func (z ZeroPost) GetID() string {
-	return z.ID
+func (z ZeroPost) GetID() Identifier {
+	return Identifier{ID: z.ID, LID: z.LID, Name: "zero-posts"}
```

Comment:
No need to define Name, since GetName was not implemented before

Reply:
removed